### PR TITLE
[7] Tools ign gz

### PR DESCRIPTION
**Pull request by nkoenig**
Changes all references to ignition in the `tools` directory.

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: tools/versions.md [hunk 1]
```diff
@@ -40,37 +40,37 @@ How to read the columns:
 ||6|EOL ❌|||F||E|2020-03-31|2022-03 with Edifice|
 ||7|stable|||||F|2021-09-30|2026-09 with Fortress|
 ||8|not-released|||||G|2022-09|TBD|
-|ignition-gazebo|1|EOL ❌||B|||A|2019-03-02|2019-09 with Acropolis|
+|gz-gazebo|1|EOL ❌||B|||A|2019-03-02|2019-09 with Acropolis|
```
Comment (methylDragon) [inline tools/versions.md:43]: Beware!

File: tools/Dockerfile.fortress [hunk 2]
```diff
@@ -20,70 +20,70 @@ RUN scripts/install_common_deps.sh
...
-      $IGN_VERSION_DATE \
-      $IGN_VERSION_PASSWORD; exit 0
+RUN scripts/build_gz.sh gazebosim gz-physics ign-physics5 y \
+      $GZ_VERSION_DATE \
+      $GZ_VERSION_PASSWORD; exit 0
 
-RUN scripts/build_ign.sh ignitionrobotics ign-gazebo ign-gazebo6 y \
-      $IGN_VERSION_DATE \
-      $IGN_VERSION_PASSWORD; exit 0
+RUN scripts/build_gz.sh gazebosim gz-gazebo ign-gazebo6 y \
```
Comment (methylDragon) [inline tools/Dockerfile.fortress:83]: Beware!

Comment (nkoenig) [reply tools/Dockerfile.fortress]: [661afce](https://github.com/gazebosim/docs/pull/262/commits/661afce09add236358e553c391fbaa1730c697b9)

File: tools/versions.md (hunk 1 above)
Comment (nkoenig) [reply tools/versions.md]: [661afce](https://github.com/gazebosim/docs/pull/262/commits/661afce09add236358e553c391fbaa1730c697b9)

**Review by methylDragon — APPROVED** [review]

**Pull request merged**